cypress/integration/patch/subscription-modal.ts

```diff
@@ -45,7 +45,7 @@ describe("Version Subscription Modal", () => {
       cy.dataTestId("1-display-name-option").should(
         "have.css",
         "user-select",
-        "none"
+        "auto"
```

Comment:
we should probably mute or adjust this test since checking css is no longer a reliable way to test if the item is disabled